src/consensus/pbft/libbyz/prepare.cpp

```diff
@@ -56,7 +62,6 @@ Prepare::Prepare(View v, Seqno s, Digest& d, Principal* dst, bool is_signed) :
   rep().prepare_sig_size = 0;
 #endif
 
-  rep().id = pbft::GlobalState::get_node().id();
```

Comment:
without this line is rep().id not set if if >=0?

Reply:
id has nothing to do with `f`, please see my comments above